contrib/pkginstaller/Distribution.in

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8" standalone="no"?>
+<installer-script minSpecVersion="1.000000">
+    <title>Podman __VERSION__</title>
+    <background mime-type="image/png" file="banner.png" scaling="proportional"/>
+    <welcome file="welcome.html" mime-type="text/html" />
+    <conclusion file="conclusion.html" mime-type="text/html" />
+    <license file="LICENSE.txt"/>
+    <options customize="never" />
```

Comment:
```suggestion
    <options customize="always" hostArchitectures="x86_64,arm64"/>
```

Reply:
only adding `hostArchitectures="x86_64,arm64"` solves the rosetta problem, the `customize="always"` attribute shows a component selection page with only one option to select so keeping it as `customize="never"`